dms/models/directory.py

```diff
@@ -48,23 +49,17 @@ class DmsDirectory(models.Model):
         parent inherits the settings form its parent.""",
     )
 
-    root_storage_id = fields.Many2one(
-        comodel_name="dms.storage",
-        string="Root Storage",
-        ondelete="restrict",
-        compute="_compute_directory_type",
-        store=True,
-        readonly=False,
-        copy=True,
-    )
     # Override acording to defined in AbstractDmsMixin
     storage_id = fields.Many2one(
-        compute="_compute_storage",
+        compute="_compute_storage_id",
+        compute_sudo=True,
+        readonly=False,
         comodel_name="dms.storage",
         string="Storage",
         ondelete="restrict",
-        auto_join=True,
```

Comment:
Why do you need to remove it?

Reply:
Do you remember these change @pedrobaeza ?
IMO these change is needed because we have added `default=False` right?

Reply:
I think it's a mistake, as they are not related.

Reply:
So keep it please